src/main/java/teammates/ui/controller/InstructorSearchPageAction.java

```diff
@@ -32,8 +32,8 @@ protected ActionResult execute() {
             numberOfSearchOptions++;
         }
 
-        boolean isSearchCommentForResponses = getRequestParamAsBoolean(Const.ParamsNames.SEARCH_COMMENTS_FOR_RESPONSES);
-        if (isSearchCommentForResponses) {
+        boolean isSearchFeedbackSessionData = getRequestParamAsBoolean(Const.ParamsNames.SEARCH_FEEDBACK_SESSION_DATA);
```

Comment:
Since we have changed this, it also makes sense to rename the variables in `InstructorSearchPageData.java`

Here:
https://github.com/TEAMMATES/teammates/blob/a88f169d8b34e1e50317ac91306aa015bc68b1ba/src/main/java/teammates/ui/pagedata/InstructorSearchPageData.java#L37-L41